### ISSUE TITLE
Linux Flatpak support

### ISSUE THREAD

**Issue by UnclePsymon** [LykosAI/StabilityMatrix#1053]
### Proposal

Flatpak is a universal packaging format for Linux distributions. I know that there is already an appimage, but they do have some issues on my Linux distro NixOS and some others as well I believe.

Maybe you could publish StabilityMatrix as a Flatpak on https://flathub.org?
Thx!

**Comment by TGlide** [comment]
bump, or a way to run it on NixOS in general would be great

**Comment by linuxkernel94** [comment]
I hope we can have a flatpak version soon

**Comment by mpicard** [comment]
Friendly bump

**Comment by Joelanglo** [comment]
I agree, I would love a flatpak so I could manage it declaratively. I did get it working using distrobox and arch.

**Comment by spacexcs** [comment]
It would be great if Stability Matrix could be packaged as a Flatpak and published on Flathub.

This would make installation and updates easier across Linux distributions. Thank you for considering this!
